docker/README.md

```diff
@@ -44,5 +70,5 @@ A few points to consider:
 * The name `orion1` can be anything and doesn't have to be related to the name given to the docker image in the previous section.
 * `--link mongodb:mongodb` assumes there is a docker container running a MongoDB image in your system, whose name is `mongodb`. In case you need one type `docker run --name mongodb -d mongo:2.6`.
 * In `-p 1026:1026` the first value represents the port to listen in on localhost. If you wanted to run a second context broker on your machine you should change this value to something else, for example `-p 1027:1026`.
-* Anything after the name of the container image (in this case `orion`) is interpreted as a parameter for the Orion Context Broker. In this case we are telling the broker where the MongoDB host is, represented by the name of our other MongoDB container. Take a look at the [documentation](../.../.../doc/admin/cli.md) for other command-line options.
+* Anything after the name of the container image (in this case `orion`) is interpreted as a parameter for the Orion Context Broker. In this case we are telling the broker where the MongoDB host is, represented by the name of our other MongoDB container. Take a look at the [documentation](https://github.com/telefonicaid/fiware-orion/blob/develop/doc/manuals/admin/cli.md) for other command-line options.
```

Comment:
The reason is that this doc is also automatically posted on docker hub, where the relative paths don't make sense.